recipes/pynisher/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pynisher" %}
+{% set version = "0.6.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e4e1d9366fc4ca60b4b2354b6d12e65600600a8c7bf4392c84f2f4ff4abc85ff
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - docutils >=0.3
+    - psutil
+    - python >=3.6
+    - setuptools
```

Comment:
Does this need `setuptools` at run time?

Reply:
Not 100% sure. It was extracted by Grayskull from here: https://github.com/automl/pynisher/blob/0077d7a9b9bd73a729d01bc9da0d9f01d173caa0/setup.py#L20 (I assume)

Reply:
Original grayskull meta.yaml:

```yaml
{% set name = "pynisher" %}
{% set version = "0.6.3" %}


package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pynisher-{{ version }}.tar.gz
  sha256: e4e1d9366fc4ca60b4b2354b6d12e65600600a8c7bf4392c84f2f4ff4abc85ff

build:
  number: 0
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv

requirements:
  host:
    - pip
    - python
  run:
    - docutils >=0.3
    - psutil
    - python
    - setuptools

test:
  imports:
    - pynisher
  commands:
    - pip check
  requires:
    - pip

about:
  home: https://github.com/automl/pynisher
  summary: A small Python library to limit the resources used by a function by executing it inside a subprocess.
  license: MIT
  license_file: LICENSE

extra:
  recipe-maintainers:
    - ADD_YOUR_GITHUB_ID_HERE
```

Reply:
Yep looks like it. Usually this is needed when `pkg_resources` is used in the library, but didn't see it used. Though agree since they listed it, we should include it.

Reply:
Ahh ok, TIL!

Reply:
@mfeurer Would you mind having a quick look at this discussion? Just to make sure that we're not including something that's not needed.

Reply:
Sorry, I don't know about that as I haven't written the actual package.

Reply:
I don't think it's required. It should probably be under `setup_requires`? I'm also not sure why docutils is a requirement (do you see a reason for this?) and why the information that psutil is required is duplicated with requirements.txt.

Reply:
@jakirkham Is the expert here so it's probably better if he has the final say but yes, `setuptools` is not used anywhere else (https://github.com/automl/pynisher/search?q=setuptools) so I'd assume that it's only needed for the setup.
`docutils` is only listed in the requirements (https://github.com/automl/pynisher/search?q=docutils) so it might be a leftover from a previous version.
psutil being in `requirements.txt` and `setup.py` is probably just a duplicate. Maybe there was a philosophy change. 
`Blame` says that it was added here: https://github.com/automl/pynisher/commit/a3e7ccce7b8075b058ae16b9654b91513f9d2037 so it might be good to double-check with the author of the commit.
In the end, this is only a guess though as I'm not sure if one of these tools was needed at some point and what the philosophy behind adding it was, or if they are used in a context that's not obvious when just examining the repo.

Reply:
Thanks for the explanation. We'll have a look into cleaning this up. Will new versions automatically be picked up by conda forge and the recipe automatically be updated or does that have to be done manually?

Reply:
Happy to help! New versions are generally picked up by a bot and proposed through a PR (Similar to https://github.com/conda-forge/pynisher-feedstock/pull/12) but I am watching all upstream repos of the feedstocks I'm maintaining and can then update them manually if there's a problem, so feel free to release any changes you might have. 

Also see: https://conda-forge.org/docs/maintainer/updating_pkgs.html#pushing-to-regro-cf-autotick-bot-branch

Reply:
Thanks a lot for your explanation and the suggestions. There should be a new release soonish which is a bit cleaner here.

Reply:
Awesome, happy to help! :)